packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -12,6 +13,16 @@ const {
   responsiveResolution,
 } = require(`gatsby-plugin-sharp`)
 
+const ImageFormatType = new GraphQLEnumType({
+  name: `ImageFormat`,
+  values: {
+    default: { value: `` },
```

Comment:
One more minor change. Could you make this "no change" (or something similar indicating the default is to keep the image format the same).